src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -118,6 +123,27 @@ export default (accessToken, userID, opts) => {
       {},
       { method: "PUT" }
     ),
+    dislikeArtworkLoader: gravityLoader(
+      (id) => `collection/disliked-artwork/artwork/${id}`,
+      {},
+      { method: "POST" }
+    ),
+    dislikedArtworkLoader: trackedEntityLoaderFactory(
```

Comment:
What's `trackedEntityLoaderFactory` do?

Reply:
it produces a loader that checks if an entity is followed/saved by batching API requests... but now I'm wondering if this won't actually work for dislikes actually

Reply:
if its just a data loader that bundles batch API requests then its fine but if its specifically for follows and saves.... dunno

Reply:
gotta look into this more! glad this PR is in WIP for now while I work out the kinks

Reply:
so it looks like this loader just bundles API requests and doesn't do much outside of making requests for checks on saves or follows more performant. I've tested this out and it seems fine, so I'm going to keep using it.

Reply:
Sounds good - it follows existing patterns!